src/client.zig

```diff
@@ -148,13 +166,29 @@ pub const Client = struct {
             .operation = operation,
             .message = message.ref(),
         }) catch {
```

Comment:
At present, we assume that `push()` will only return one error.

But we should explicitly switch on the error we catch here and rather show our assumption with a single error prong in the switch. That way, if more errors are added, the compiler will tell us and fail to compile.